### DISCUSSION TITLE
Problem running some Uno Platform samples

### DISCUSSION THREAD

**Discussion by jldearmas** [unoplatform/uno#15033] [Q&A]
I create a development environment for Uno Platform:

- Windows 10 22H2
- Visual Studio 2022 17.8.4
- Install Uno Platform Templates
- Install and Run Uno.Check

Then I cloned  locally Uno Platform samples and try to run:

- UnoMaterialSample
- UnoMaterialToolkitSample

But build process for both of them failed. 

![image](https://github.com/unoplatform/uno/assets/60520217/10c79ae3-12bb-4a32-8532-5e0c7fd5454c)

Even I removed from Windows project arm from platforms list, but got same result.

Do I need to install something else? 
Do I need to update/modify something in the projects?

By the way I doesn't have installed the UWP workload in VS 2022. Is it necessary?

**Comment by limeniye** [comment]
Hello, @jldearmas.
Try to change [this line](https://github.com/unoplatform/Uno.Samples/blob/8187a2182ed8167b8f42dcd9a432c205259413d3/UI/UnoMaterialSample/UnoMaterialSample/UnoMaterialSample.Windows/UnoMaterialSample.Windows.csproj#L8):
```<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>``` to this ```<RuntimeIdentifiers>win-x86;win-x64;win-arm64</RuntimeIdentifiers>``` wherever you find it.

@nickrandolph

**Reply by jeromelaban** [reply]
We're looking into this with https://github.com/unoplatform/Uno.Samples/pull/625, there were fixes in the latest WinUI bits.
